src/main/java/net/mcreator/ui/workspace/WorkspacePanelVariables.java

```diff
@@ -138,6 +139,9 @@
 						return new SpinnerCellEditor(spinner);
 					} else if (variableType == VariableTypeLoader.BuiltInTypes.LOGIC) {
 						return new DefaultCellEditor(new JComboBox<>(new String[] { "true", "false" }));
+					} else if (variableType == VariableTypeLoader.BuiltInTypes.DIRECTION) {
+						return new DefaultCellEditor(new JComboBox<>(new String[] { "Direction.DOWN", "Direction.UP",

```

Comment:
Initial values don't use mappings so direction can't be directly specified here.

There are two options:

1.  Remove initial value similar how itemstacks don't have one
2. Use mapping system in this dropdown and set initial value with mappings